openni_description/test/test_openni_desc.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+PKG = 'openni_description'
+
+import os
+import subprocess
+import unittest
+
+import rospkg
+import xacro
+
+
+class TestOpenniDescription(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        True
+
+    def test_urdf_turtlebot(self):
+        """
+        Check if check_urdf command passes with the urdf that is generated in
+        the .test file this test case is called from.
+        """
+        resulted_urdf_file_relpath = "./sample_kobuki.urdf"
+        kobuki_xacro_file_path = rospkg.RosPack().get_path('openni_description') + "/test/sample_kobuki.urdf.xacro"
+        self.assertTrue(os.path.isfile(kobuki_xacro_file_path)
+        xacro_output_memory = xacro.process_file(kobuki_xacro_file_path)
```

Comment:
This method is only available after `xacro` 1.10.x that is after ROS Jade as of today.